docs/pages/index.md

```diff
@@ -1,4 +1,8 @@
-![lodestar logo](assets/lodestar_icon_text_black_stroke.png)
+---
+title: Home
```

Comment:
`Home | Lodestar` is showing up in the tab when rendered.  Is a mix of this and the Lodestar from `title: "Lodestar"`  in the config.  Can we remove one or the other so it just shows Lodestar... or perhaps "Lodestar Documentation".  I dont feel strongly about it but "Home" feels funny as the first thing in the browser tab and it will likely also affect SEO

Reply:
Apparently can be done, but requires a bit of coding so I'd rather do that in a separate PR. See https://github.com/facebook/docusaurus/discussions/6640